tests/core/training/story_reader/test_common_story_reader.py

```diff
@@ -224,11 +223,11 @@ async def test_load_multi_file_training_data(
         hashed.append(json.dumps(sts + acts, sort_keys=True))
     hashed = sorted(hashed, reverse=True)
 
-    data = featurizer.featurize_trackers(trackers, default_domain)
-
-    featurizer_mul = MaxHistoryTrackerFeaturizer(
-        BinarySingleStateFeaturizer(), max_history=2
+    data, label_ids = featurizer.featurize_trackers(
+        trackers, default_domain, interpreter=RegexInterpreter()
     )
+
```

Comment:
this test function seems very weird. I think it makes more sense to
1. Just check that the events were correctly parsed 
2. Do any featurization tests in the test modules for the featurizers